TreasureBound/TreasureBound/PuzzleSwipeViewController.swift

```diff
@@ -135,6 +147,44 @@ class PuzzleViewController: UIViewController {
         timerLabel.text = String.init(format: "%02d", timeCount)
     }
     
+    override func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
+        let myTouch : UITouch = touches.first!
+        
+        if(blocksArr.contains(myTouch.view as Any)){
+            
+            let touchView: MyLabel = (myTouch.view)! as! MyLabel
+            
+            let xDif: CGFloat = touchView.center.x - empty.x
+            let yDif: CGFloat = touchView.center.y - empty.y
+            
+            let distance : CGFloat = sqrt(pow(xDif, 2) + pow(yDif, 2))
```

Comment:
Avoid using magic numbers "2" in this case. Use a constant and give it a name that provides context. Makes the code much more readable.